packages/http-client-csharp/emitter/src/emitter.ts

```diff
@@ -78,13 +78,7 @@ export async function $onEmit(context: EmitContext<CSharpEmitterOptions>) {
         enums: new Map(),
       },
     };
-    if (
-      context.program.diagnostics.length > 0 &&
-      context.program.diagnostics.filter((digs) => digs.severity === "error").length > 0
-    ) {
-      logDiagnostics(context.program.diagnostics, context.program.host.logSink);
-      process.exit(1);
-    }
```

Comment:
we did not need to exit the program - typespec compiler will do that after the invocation of the emitter is done.